superset/tasks/schedules.py

```diff
@@ -705,6 +722,21 @@ def run_alert_query(
     )
     db.session.commit()
 
+
+def validate_observations(alert_id: int, label: str) -> Optional[str]:
+    """
+    Runs an alert's validators to check if it should be triggered or not
+    If so, return the name of the validator that returned true
+    """
+
+    logger.info("Validating observations for alert <%s:%s>", alert_id, label)
+
+    alert = db.session.query(Alert).get(alert_id)
+    for validator in alert.alert_validators:
```

Comment:
can there be multiple ?

Reply:
currently yes

Reply:
hm, this is a deviation from design, any specific reason to do that?